test/tarantool/cpu_metrics_test.lua

```diff
@@ -14,6 +14,8 @@ end)
 
 g.test_cpu = function(cg)
     cg.server:exec(function()
+        t.skip('Flaky test')
```

Comment:
It seems a good idea to create a new issue/add a link to the issue and fix the problem later.

Reply:
Added #492